### PR TITLE
Update openconnect package to version 8.20

### PR DESCRIPTION
**Pull request by jamuanfc**
update openconnect package to 8.20 

### Run the following to get this pull request's changes locally for testing.
```
CREW_TESTING_REPO=https://github.com/jamuanfc/chromebrew.git CREW_TESTING_BRANCH=update-package  CREW_TESTING=1 crew update
```

### CODE REVIEW

**Comment by uberhacker** [comment]
Thank you for the update @jamuanfc!  Are you planning on building the binaries @satmandu?

File: packages/openconnect.rb [hunk 1]
```diff
@@ -30,32 +18,35 @@ class Openconnect < Package
...
-      system "chmod +x vpnc-start"
-      system "echo '#!/bin/bash' > vpnc-stop"
-      system "echo 'killall openconnect' >> vpnc-stop"
-      system "echo 'sudo ip tuntap del mode tun tun0' >> vpnc-stop"
-      system "chmod +x vpnc-stop"
-    end
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin"
+    @vpnc_start = <<~'VPNC_STARTEOF'
+      #!/bin/bash
+      if test \$1; then
```
Comment (uberhacker) [inline packages/openconnect.rb:31]: Don't need to escape?

Comment (satmandu) [reply packages/openconnect.rb]: Fixed

**Comment by satmandu** [comment]
binaries added but untested @uberhacker

**Comment by jamuanfc** [comment]
thanks, I will try..

**Comment by jamuanfc** [comment]
the new version is installed. but I can't connect to vpn server. (trying 2 different vpn server) . the error message is "Cannot find device "tun0".

any idea what happened there?

Screenshot 2022-04-01 21 12 56

Screenshot 2022-04-01 21 28 47

**Comment by satmandu** [comment]
Might need to do `sudo start shill BLACKLISTED_DEVICES="br0,tun0"` first.

**Comment by satmandu** [comment]
See https://github.com/dnschneid/crouton/issues/2215

**Comment by satmandu** [comment]
If the shill command is needed, we would have to modify `vpnc-start` ...

**Comment by jamuanfc** [comment]
Screenshot 2022-04-01 23 16 42

it works with

```
sudo stop shill
sudo start shill BLOCKED_DEVICES="tun0,br0"
```

https://github.com/dnschneid/crouton/wiki/Using-Cisco-AnyConnect-VPN-with-openconnect#solution-to-make-the-tun0-interface-stable-and-the-openvpn-command-reliable

thank you very much..

**Comment by satmandu** [comment]
ok so we can add this to the top of the config file:
```
trap '' 2
# Stop shill and restart it with a nicer attitude towards tun0
sudo stop shill
sudo start shill BLACKLISTED_DEVICES="tun0,br0"
# Sleep 10 seconds to allow ChromeOS to reconnect to the network
sleep 10
```

Let me do respin the builds...

**Comment by satmandu** [comment]
@jamuanfc please do the following and test if this works again:
```
CREW_TESTING_REPO=https://github.com/jamuanfc/chromebrew.git CREW_TESTING_BRANCH=update-package  CREW_TESTING=1 crew update
crew reinstall openconnect
```

**Comment by jamuanfc** [comment]
> ok so we can add this to the top of the config file:
> 
> ```
> # Stop shill and restart it with a nicer attitude towards tun0
> sudo stop shill
> sudo start shill BLACKLISTED_DEVICES="tun0,br0"
> # Sleep 10 seconds to allow ChromeOS to reconnect to the network
> sleep 10
> ```
> 
> Let me do respin the builds...

for me, works with.
sudo start shill BLOCKED_DEVICES="tun0,br0"

**Comment by satmandu** [comment]
ok I'll change that text in the scripts... hold on.

**Comment by satmandu** [comment]
I think this is good to go now...

**Pull request merged**